spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/headers.adoc

```diff
@@ -179,3 +179,27 @@ MessagingMessageConverter converter() {
 
 If using Spring Boot, it will auto configure this converter bean into the auto-configured `KafkaTemplate`; otherwise you should add this converter to the template.
 
+[[multi-value-header]]
+== Support multi-value header
+
+Spring for Apache Kafka 4.0 supports multi-value header.
+To use multi-value header, the patterns for multi-value should be configured to `HeaderMapper`.
+If no patterns are configured to `HeaderMapper`, iterable header values under same header name will be handled like single value.
+
+If there are use cases in which multi-value headers are required, use this:
```

Comment:
The `use this` is a command for specific person.
That is not a technical documentation sentence.

Either way, I don't find the text here as useful as it could be.
I would expect an explanation about multi-value headers in Kafka and then how their are handled by default and so on.
Then other other hand, there should be an explanation how iterable message headers are processed now and what we do if there is respective pattern matching.